src/software/ai/hl/stp/stp.cpp

```diff
@@ -155,13 +156,15 @@ PlayInfo STP::getPlayInfo()
 {
     std::string info_referee_command = toString(current_game_state.getRefereeCommand());
     std::string info_play_name = getCurrentPlayName() ? *getCurrentPlayName() : "No Play";
-    PlayInfo info              = PlayInfo(info_referee_command, info_play_name, {});
+    PlayInfo info              = PlayInfo();
```

Comment:
For ease of code navigation and consistency with other protos, I think we should have constructors for PlayInfo in `proto/message_translation`

Reply:
on second thought i dont think this is needed